### DISCUSSION TITLE
Changes in NearbyShare as a result of merging with QuickShare

### DISCUSSION THREAD

**Discussion by 0xYakuV** [google/nearby#2450] [Q&A] [answered]
Hi,
I am working on a project that relies on how the NearbyShare app works and uses the Nearby Connections API. Google and Samsung just announced in CES 2024 that Nearby Share and Quick Share are merging into one service which will be called Quick Share. Do you have any information about how the new Quick Share is going to work internally? Is the protocol about to change somehow? Is the Desktop Nearby Share app going to change somehow internally other than UI changes?
I am afraid of a scenario where I will develop something that relies on how NearbyShare works today but it will all be for nothing as it will maybe change in the next month. It will be extremely helpful if you can provide information about if and what is going to change in the protocol internally so I can prepare

**Comment by bourdakos1** [comment, marked as answer]
The Nearby Share protocol won’t be changing due to the merge, so you shouldn’t have to worry about wasted efforts :)
